dashboard/src/shared/ResourceRef.ts

```diff
@@ -41,7 +41,7 @@ class ResourceRef {
   }
 
   // Gets the plural form of the resource Kind for use in the resource path
-  private resourcePlural() {
+  public resourcePlural() {
```

Comment:
since this is now public I would add a test for checking the supported plurals so we make sure we don't break the function

Reply:
sure, I decided not to since it was already being tested in `getResourceURL()`, but will add a more specific one here.